.ci/package-storage.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Should this file be called `package-registry`?

Reply:
C&P 🤦